packages/ui/src/components/va-input/components/VaInputWrapper.vue

```diff
@@ -74,6 +74,15 @@
       </div>
     </div>
 
+    <div v-if="hasCounterOrMaxLengthComputed || $slots.hint" class="va-input-wrapper-bottom">
+      <div v-if="$slots.hint" class="va-input-wrapper-bottom__hint">
+        <slot name="hint" />
+      </div>
+      <div v-if="hasCounterOrMaxLengthComputed" class="va-input-wrapper-bottom__counter">
+        {{ $props.counterOrMaxLength }}
+      </div>
+    </div>
```

Comment:
I think it would be better to give a slot to the user here
```suggestion
    <div v-if="$props.counter" class="va-input-wrapper__counter">
      <slot name="counter" v-bind="{ counter, maxLength }">
        <span class="...">{{ `${counter}/${maxLength}` }}</span>
      </slot>
    </div>
```

Reply:
If I do this, here will empty space.

Reply:
I think slot here is will be over-engineering thing, and if we remove the hint slot, here will be an empty space that doesn't look good. :)